lib/xlat_tables_v2/xlat_tables_common.c

```diff
@@ -1,123 +0,0 @@
-/*
```

Comment:
Sorry, I've changed my mind. I forgot this was in xlat_tables_v2, which we don't guarantee compatibility for yet.